entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+function print_help {
+    echo "Available options:"
+    echo " start commands (Rasa Core cmdline arguments) - Start Rasa Core server"
+    echo " train                                        - Train a dialogue model"
+    echo " start -h                                     - Print Rasa Core help"
+    echo " help                                         - Print this help"
+    echo " run                                          - Run an arbitrary command inside the container"
+}
+
+case ${1} in
+    start)
+        exec python -m rasa_core.server "${@:2}"
+        ;;
+    run)
+        exec "${@:2}"
+        ;;
+    train)
+        exec python -m rasa_core.train "${@:2}"
```

Comment:
how does the train work because story file & domain are not part of the container nor are they mounted right?

Reply:
Yes, for training you would have to explicitly add `--stories`, `--out` and `--domain`. The files would have to be mounted into the container